egs/librispeech/ASR/local/compute_fbank_musan.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+"""
+This file computes fbank features of the musan dataset.
+Its looks for manifests in the directory data/manifests
+and generated fbank features are saved in data/fbank.
+"""
+
+import os
+import subprocess
+from contextlib import contextmanager
+from pathlib import Path
+
+from lhotse import CutSet, Fbank, FbankConfig, LilcomHdf5Writer, combine
+from lhotse.recipes.utils import read_manifests_if_cached
+
+
+@contextmanager
+def get_executor():
```

Comment:
This executor bit seems like a good candidate to move to the library-level?

Reply:
Will move it to a new file local/utils.py